src/gutter-component.coffee

```diff
@@ -15,13 +15,26 @@ GutterComponent = React.createClass
   dummyLineNumberNode: null
   measuredWidth: null
 
+  propTypes:
+    isHiddenInitially: React.PropTypes.bool
+
+  getDefaultProps: ->
+    isHiddenInitially: false
```

Comment:
> Ideally the only props any view component should have in the editor component hierarchy is the presenter.
nice

> In this case, I think you can base the visibility of the gutter component on the model for now.
I'd included this just so this component has a defined default behavior. If the visibility is for some reason not passed in (maybe by accident), this would make it visible. Don't feel too strongly about this if you'd prefer the simplicity of fewer lines.